ui/package.json

```diff
@@ -60,7 +60,7 @@
     "markdown-it-highlightjs": "^3.0.0",
     "markdown-it-link-attributes": "^3.0.0",
     "material-ui-pickers": "2.2.4",
-    "material-ui-treeview": "^3.2.0",
+    "material-ui-treeview": "^3.4.0",
```

Comment:
Ah, the renovate bot recently filed a PR to update this package to `3.5.0` in https://github.com/taskcluster/taskcluster/pull/1716. This line is probably not necessary anymore.